utils.py

```diff
@@ -37,8 +37,13 @@
 import feconf
 import python_utils
 
+from typing import ( # isort:skip # pylint: disable=unused-import, import-only-modules
```

Comment:
This looks very untidy, are we going to have these imports and disable flags in all the files?

Reply:
Yes this is required as pylint as thinks there imports are unused. Other option is to use `# noqa`. What do you prefer?

Reply:
Can we change the ImportOnlyModulesChecker to allow importing class (not modules) from `typings` and `__future__`? [You can do it in a separate PR]

Reply:
Overall solution:
- To avoid import-only-modules:  change the ImportOnlyModulesChecker to allow importing class (not modules) from typings and __future__

- To avoid unused-import: Move all typing class to `dummy-variables-rgx` in .pylintrc (Possible?)

- To avoid isort:skip: Add imports in sorted order*

Reply:
@DubeySandeep Can I postpone this a bit? I already have a lot of work pending. Thanks a lot!

Reply:
> @DubeySandeep Can I postpone this a bit? I already have a lot of work pending. Thanks a lot!

I'm fine with postponing if we are not going to add more disable flags* [Note: I think this will help to have a better python typing structure for developers to write else they will start using disable flag and it will look like we are promoting such flag usage.]

Reply:
I get your point. I'll fix this with a new PR. Here is the issue for this: https://github.com/oppia/oppia/issues/13171 . 
Note: # isort: skip tells me to split this import for each of the type. Example:
```
from typing import Any
from typing import Dict
```
So do you think it is good to import this way?

Reply:
@DubeySandeep I'll make a PR for this before I add type annotations to new files.